best-practices/rails/controllers/README.md

```diff
@@ -0,0 +1,25 @@
+## Ruby on Rails Controllers
+
+###### Restful
```

Comment:
@vgsantoniazzi use third level indentation. Follow indentation as possible in markdown.

```markdown
### Restfull

Bla bla
```